src/OrleansVSTools/OrleansVSTools/OrleansVSTools.csproj

```diff
@@ -202,8 +202,8 @@
       <Link>Packages\Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0.nupkg</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="..\packages\Newtonsoft.Json.7.0.1\Newtonsoft.Json.7.0.1.nupkg">
-      <Link>Packages\Newtonsoft.Json.7.0.1.nupkg</Link>
+    <Content Include="..\packages\Newtonsoft.Json.9.0.1\Newtonsoft.Json.9.0.1.nupkg">
+      <Link>Packages\Newtonsoft.Json.9.0.1.nupkg</Link>
```

Comment:
If we update the VSTools project (which is still 1.4.0), then we are missing updating binding redirects in the template projects (or does the host project automatically adds redirects when the template unfolds?). Same as before, either don't update these tools at all, or do the full change. It's probably easier to just revert, as we don't tend to keep the OrleansVSTools up to date until we are ready to release, but you are free to choose 👅